ProjectSidewalk/SidewalkWebpage
# 3062 - adding logging for all emails to forgot password controller via logger

Resolves #3062 

Added logging using the info level of logging to the ForgotPasswordController so all emails are logged when users are trying to reset their password on the forgot password page. (Note to self: Scala logging documentation [here](https://www.playframework.com/documentation/2.8.x/ScalaLogging))

##### Testing instructions
1. Go to "Forgot password" page and type in email
2. Check logs to confirm "Logging user's email: " + email entered is present

Future steps: If time permits, I want to explore the database option and making this an activity in the webpageactivity table. For now this is just to get the ticket resolved but this can be innovated in the future for sure.

##### Things to check before submitting the PR 
- [TBD] I've updated any logging. Clicks, keyboard presses, and other user interactions should be logged. If you're not sure how (or if you need to update the logging), ask Mikey. Then make sure the documentation on [this wiki page](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Descriptions-of-Logged-Events) is up to date for the logs you added/updated.
